pywbem/cim_obj.py

```diff
@@ -5840,7 +5822,7 @@ def cimvalue(value, type):
 
     if type is None:
         # The following may raise TypeError or ValueError. This enforces that
-        # Python integer or float values cannot be provided without specifing
+        # Python number typed values cannot be provided without specifing
```

Comment:
Change word in line above from enforces to ensures

Reply:
DONE